scripts/build-windows-enterprise-installer.ps1

```
@@ -0,0 +1,416 @@
+    )
+    [void]$acl.AddAccessRule($rule)
+    Set-Acl -LiteralPath $Path -AclObject $acl
+}
+
+function Copy-ZipEntry {
+    param(
+        [Parameter(Mandatory)]$Archive,
+        [Parameter(Mandatory)][string]$EntryName,
+        [Parameter(Mandatory)][string]$Destination
+    )
+    $matches = @($Archive.Entries | Where-Object {
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'matches' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)